modules/ocf_mirrors/manifests/init.pp

```diff
@@ -178,6 +178,13 @@
     special => 'daily',
   }
 
+  file { '/opt/mirrors/bin/health':
+    source  => 'puppet:///modules/ocf_mirrors/health',
```

Comment:
this is only for debian-style healthchecks, can you call this `debian-healthcheck` or something?

Reply:
In regards to this and the comment below about the type param, I think ```ftpsync-healthcheck``` is probably the best name.

Reply:
i think `ftpsync-healthcheck` is a bit of a misnomer; ftpsync is just how we download the files, you could use rsync directly or `wget -r` or ftp to download the mirror contents, too. the healthcheck would work on any debian-compatible repo.